### PR TITLE
Personal portfolio review branch Pull request

### PR DESCRIPTION
**Pull request by Brian3Kamau**
A review of Personal Portfolio webpage project from the Microverse trials

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          <a href="">
+            <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/President_Barack_Obama.jpg/1200px-President_Barack_Obama.jpg" alt="Official portrait. 2012" class="project-img"/>
+            <p class="project-tile">The Story of President Barrack Obama</p>
+          </a>
+          <a href="">
+            <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d7/US_President_Barack_Obama_taking_his_Oath_of_Office_-_2009Jan20.jpg/1200px-US_President_Barack_Obama_taking_his_Oath_of_Office_-_2009Jan20.jpg" alt="US President Barack Obama taking his Oath of Office - 2009Jan20.jpg" class="project-img"/>
+            <p class="project-tile">President Barack Obama taking his Oath of Office</p>
+          </a>
+          
+        </div>
```
Comment (Brian3Kamau) [inline index.html:37]: - [ ] Please do not leave a blank space in this block of code. The blank space in this block of code is unnecessary. To make your code cleaner and easier to read, it is better to leave space only between complete blocks of code, and not within each block.

File: index.html [hunk 2]
```diff
@@ -26,11 +26,11 @@ <h1>Hey, We are Brian and Michael</h1>
       <h2>We are currently working on this Project</h2>
         <div class="project-grid">
           <a href="">
-            <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/President_Barack_Obama.jpg/1200px-President_Barack_Obama.jpg" alt="Official portrait. 2012" class="project-img"/>
+            <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/President_Barack_Obama.jpg/1200px-President_Barack_Obama.jpg" alt="" class="project-img"/>
             <p class="project-tile">The Story of President Barrack Obama</p>
           </a>
           <a href="">
-            <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d7/US_President_Barack_Obama_taking_his_Oath_of_Office_-_2009Jan20.jpg/1200px-US_President_Barack_Obama_taking_his_Oath_of_Office_-_2009Jan20.jpg" alt="US President Barack Obama taking his Oath of Office - 2009Jan20.jpg" class="project-img"/>
```
Comment (Brian3Kamau) [inline index.html]: Please ensure that each of your image tags has an alt attribute. Alt attributes are important, especially for visually impaired people who depend on screen readers  to access your webpage.

**Review by Brian3Kamau — COMMENTED** [review]
Hi @Brian3Kamau, you have made a great effort in working on your Project. 👍 

However there are a few changes that you could implement to make your project much better, and professional.  Please take a look at the following:- 

### Highlights :
- Proper use of CSS Selectors ✔️ 
- Proper use of HTML Semantic tags ✔️ 

### Required Changes ♻️ 
1. Check the Comments under the Review
2. Use Professional README.md. Use the Microverse  README Template and customize it according to your personal information. 
3. Use the Correct git flow in your project.  

Happy coding 👏

File: index.html (hunk 1 above)
Comment (Ibizugbe) [reply index.html:37]: [OPTIONAL] Your comment looks polite and you have explained to the student why you have requested the change👍👍 But I think you can make this better by adding a more descriptive reason for requesting a change😉 You can furher explain why the block of code is unncessary to make your comment clearer and more descriptive👍

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Brian3Kamau 👋👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [ ] Atleast two comments have been written using the github inline comment feature👍.
- [ ] Suggestions have been phrased politely👍.


## Required Changes ♻️

- [ ] You currently haven't made use of the `review changes` feature🥲 If this is done all your comments will be placed under your summary and this gives your review a more organised structure👍 and makes it easier to understand👍 kindly study [this lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) and take note of the [las gif image](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/images/images_finish_review.gif?raw=true)👍. as described in the `gif` you should click the `start review`  when you highlight the block of code you want to add an inline comment so as to position your comments under your summary😉

_Check the comments under the review._

## Optional suggestions
- [OPTIONAL] We can make your comments better by adding emojis as they give your comments a more humanly friendly perception😉👍

_see comments below_

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+          <a href="">
+            <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/President_Barack_Obama.jpg/1200px-President_Barack_Obama.jpg" alt="" class="project-img"/>
+            <p class="project-tile">The Story of President Barrack Obama</p>
+          </a>
+          <a href="">
+            <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d7/US_President_Barack_Obama_taking_his_Oath_of_Office_-_2009Jan20.jpg/1200px-US_President_Barack_Obama_taking_his_Oath_of_Office_-_2009Jan20.jpg" alt="" class="project-img"/>
+            <p class="project-tile">President Barack Obama taking his Oath of Office</p>
+          </a>
+          
+        </div>
```
Comment (Brian3Kamau) [inline index.html:37]: Please do not leave a blank space in this block of code. The blank space in this block of code is unnecessary. To make your code cleaner and easier to read, it is better to leave space only between complete blocks of code, and not within each block.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    </section>
+    <section id="projects" class="projects">
+      <h2>We are currently working on this Project</h2>
+        <div class="project-grid">
+          <a href="">
+            <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/President_Barack_Obama.jpg/1200px-President_Barack_Obama.jpg" alt="" class="project-img"/>
+            <p class="project-tile">The Story of President Barrack Obama</p>
+          </a>
+          <a href="">
+            <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d7/US_President_Barack_Obama_taking_his_Oath_of_Office_-_2009Jan20.jpg/1200px-US_President_Barack_Obama_taking_his_Oath_of_Office_-_2009Jan20.jpg" alt="" class="project-img"/>
```
Comment (Brian3Kamau) [inline index.html:33]: - [ ] Please ensure that each of your image tags has an alt attribute. Alt attributes are important, especially for visually impaired people who depend on screen readers /to access your webpage.

**Review by Brian3Kamau — COMMENTED** [review]
Hi @Brian3Kamau, you have made a great effort in working on your Project. 👍 

However there are a few changes that you could implement to make your project much better, and professional.  Please take a look at the following:- 

### Highlights :

- [ ] Proper use of CSS Selectors ✔️ 

- [ ] Proper use of HTML Semantic tags ✔️ 

- [ ] Your project runs on my machine ✔️ 


### Required Changes ♻️ 

- [ ]  Check the Comments under the Review

- [ ] Use Professional README.md. Use the Microverse  README Template and customize it according to your personal information. 

- [ ]  Use the Correct git flow in your project.  

Happy coding 👏

**Review by Owusu-Desmond — APPROVED** [review]
Hi @Brian3Kamau , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 Descriptive pull request✅
📌 Good commit messages ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**